language/polkavm/move-to-polka/tests/common/mod.rs

```diff
@@ -58,30 +43,14 @@ impl BuildOptions {
     }
 }
 
-pub fn build_move_program(options: BuildOptions) -> anyhow::Result<Vec<u8>> {
+pub fn build_polka_from_move(options: BuildOptions) -> anyhow::Result<Vec<u8>> {
+    let output_file = options.options.output.clone();
     // parse move source files
     let mut color_writer = create_colored_stdout();
-    let move_env = get_env_from_source(&mut color_writer, &options.options)?;
-    let output_file = &options.output_file;
-    println!("Object file output -> {output_file}");
-    create_dir_all(output_file)?;
-    // translate to riscV object file
-    let mut llvm_translate_options = Options::default();
-    llvm_translate_options.compile = true; // we don't need linking stage
-    llvm_translate_options.output = output_file.to_string();
-    llvm_translate_options.llvm_ir = false;
-    compile(&move_env, &llvm_translate_options)?;
+    run_to_polka(&mut color_writer, options.options)?;
 
     //TODO it would be so nice if compile won't access FS directly so we can work purely in-memory
     let data = std::fs::read(output_file)?;
     Ok(data)
 }
 pub const MOVE_STDLIB_PATH: &str = env!("MOVE_STDLIB_PATH");
```

Comment:
Please move this declaration to the top of the module. I wanted to do this still in my PR, but you merged it already 😁

Reply:
Done!